.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
     "extends": "./Tools/eslint-config-cesium/browser.js",
+    "globals": {
+        "defineSuite": true
```

Comment:
Since this global only applies to the tests, it should be in `Specs/.eslintrc.json` only.